src/pages/TCY/types.ts

```diff
@@ -0,0 +1,27 @@
+export enum TCYTabIndex {
+  Stake = 0,
+  Unstake = 1,
+  Claim = 2,
+  ChangeAddress = 3,
+}
```

Comment:
Nitpick: TS assigns the numbers implicitly, so you can omit them if you'd like.

```suggestion
export enum TCYTabIndex {
  Stake,
  Unstake,
  Claim,
  ChangeAddress,
}
```